specs/driver_spec.rb

```diff
@@ -12,7 +13,12 @@
     end
 
     it "throws an argument error with a bad ID value" do
-      proc{ RideShare::Driver.new(id: 0, name: "George", vin: "33133313331333133")}.must_raise ArgumentError
+      begin
+        RideShare::Driver.new(id: 0, name: "George", vin: "33133313331333133")
+        fail
```

Comment:
I like the exploration here, but in general it's preferable to use the built-in `must_raise` method.